pkg/role_binding/role_binding.go

```diff
@@ -119,10 +128,32 @@ func PresentRoleBinding(rb *v1.RoleBinding) RoleBinding {
 	}
 }
 
-func PresentRoleBindings(roleBindings []*v1.RoleBinding) []string {
-	var roleList []string
+func PresentRoleBindings(roleBindings []*v1.RoleBinding) []AccountRoleBinding {
+	keySeparator := "__"
+	uniqueRoleBindingsMap := make(map[string]int)
+	var uniqueRoleList []AccountRoleBinding
 	for _, roleBinding := range roleBindings {
-		roleList = append(roleList, roleBinding.Role().ID())
+		if val, ok := uniqueRoleBindingsMap[roleBinding.Role().ID()+keySeparator+roleBinding.Type()]; ok {
+			uniqueRoleBindingsMap[roleBinding.Role().ID()+keySeparator+roleBinding.Type()] = val + 1
+		} else {
+			uniqueRoleBindingsMap[roleBinding.Role().ID()+keySeparator+roleBinding.Type()] = 1
+		}
+	}
+	for k := range uniqueRoleBindingsMap {
+		keySegments := strings.Split(k, "__")
+		var totalOccs *int
```

Comment:
I'm not sure why we need to use a pointer instead of the int type.

Reply:
TotalOccurrences is a pointer so that it can be set to nil, which would eventually not be shown as a field when the count is 1